DescriptionFeatureApp/DescriptionFeatureAppApp.swift

```diff
@@ -9,7 +9,7 @@ struct DescriptionFeatureAppApp: App {
         store: .init(
           initialState: .init(),
           reducer: descriptionReducer,
-          environment: DescriptionEnvironment()
+          environment: DescriptionEnvironment(backgroundQueue: .main)
```

Comment:
this I don't get? I saw its just the feature test app but could you explain?

Reply:
Without this, when you select the target DescriptionFeatureApp and try to run you see a compiler error: "Missing argument for parameter backgroundQueue in call". So I added it. 🤷‍♂️